content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,228 @@
+    ```
+
+    The response should be something like this:
+
+    ```
+    Python 3.6.5
+    ```
+
+    If you do not have Python 3.6 or later, you can [download
+    Python](https://www.python.org/downloads/) from the Python Software
+    Foundation.
+
+1.  Use virtualenv to create a virtual environment to install Kubeflow
+    Fairing in. To check if you have virtualenv installed, run the
+    following command: 
+
+    ```bash
+    which virtualenv
+    ```
+
+    The response should be something like this.
+
+    ```bash
+    /usr/bin/virtualenv
+    ```
+
+    If you do not have virtualenv, use pip3 to install virtualenv.
+
+    ```bash
+    pip3 install --upgrade virtualenv
+    ```
+
+    Create a new virtual environment, and activate it.
+
+    ```bash
+    virtualenv venv --python=python3
+    source venv/bin/activate
+    ```
+
+1.  Install Jupyter Notebook.
+
+    ```bash
+    pip3 install --upgrade jupyter
+    ```
+
+1.  Install Kubeflow Fairing from the cloned repository.
+
+    ```bash
+    pip3 install --upgrade .
+    ```
+
+1.  Install the Python dependencies for the XGBoost demo notebook.
+
+    ```bash
+    pip3 install -r examples/prediction/requirements.txt
+    ```
+
+## Install and configure the Google Cloud SDK
+
+In order to use Kubeflow Fairing to train or deploy to Kubeflow on Google
+Kubernetes Engine, or Google Cloud Machine Learning Engine, you must configure
+your development environment with access to Google Cloud Platform. 
+
+1.  If you do not have the Google Cloud SDK installed, [install the Google
+    Cloud SDK][gcloud-install].
+
+1.  Use `gcloud` to set a default project.
+
+    ```bash
+    export PROJECT_ID=<your-project-id>
+    gcloud config set project $PROJECT_ID
+    ```
+
+1.  Kubeflow Fairing needs a service account to make API calls to Google Cloud
+    Platform (GCP). The recommended way to provide Fairing with access to this
+    service account is to set the GOOGLE_APPLICATION_CREDENTIALS environmental
+    variable. To check for the GOOGLE_APPLICATION_CREDENTIALS environmental
+    variable, run the following command:
+
+    ```bash
+    ls "$GOOGLE_APPLICATION_CREDENTIALS"
+    ```
+
+    The response should be something like this:
+
+    ```bash
+    /.../.../key.json
+    ```
+
+    If you do not have a service account, then let's create one and grant it
+    access to the required roles.
+
+    ```bash
+    export SA_NAME=<your-sa-name>
+    gcloud iam service-accounts create $SA_NAME
+    gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com  --role 'roles/editor'
+    ```
+
+    Create a key for your service account.
+
+    ```bash
+    gcloud iam service-accounts keys create ~/key.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
+    ```
+
+    Create the GOOGLE_APPLICATION_CREDENTIALS environmental variable.
+
+    ```bash
+    export GOOGLE_APPLICATION_CREDENTIALS=~/key.json
+    ```
+
+## Docker setup
```

Comment:
s/Docker setup/Set up Docker

(To make the headings parallel.)

Reply:
done